protocol-units/settlement/mcr/config/src/lib.rs

```diff
@@ -22,16 +22,17 @@ pub struct Config {
 	#[serde(default = "default_mcr_contract_address")]
 	pub mcr_contract_address: String,
 	#[serde(default = "default_gas_limit")]
-	pub gas_limit: u128,
+	pub gas_limit: u64,
 	#[serde(default = "default_tx_send_retries")]
 	pub tx_send_retries: u32,
+	pub anvil_process_pid: Option<u32>,
```

Comment:
Can we make management of infrastructure processes part of the orchestration layer, without adding dynamic warts to the config?

If this field is present in the config on file, but the process has already finished and possibly another one has taken its PID, this can get messy.

Reply:
Filed this concern as #143 to unblock this PR.

Reply:
It's for local testing in a managed env. If anvil stop it won't be restarted unless the user explicitly do it.